backend/main.py

```diff
@@ -40,7 +40,7 @@
     allow_headers=["*"],
 )
 
-if "pytest" not in sys.modules:
+if "pytest" not in sys.modules and not DISABLE_CSRF_PROTECTION:
```

Comment:
Users are given the option to disable CSRF protection if it causes issues

Reply:
Should we add it to the docker-compose example?

Reply:
no it'll go in https://github.com/zurdi15/romm/wiki/Environment-Variables

Reply:
ahh yes, forgot about it! good one wiki page